app/views/welcome/index.html.slim

```diff
@@ -4,6 +4,7 @@
 
 h3
   = t(".members")
+  .badge =< @members.size
```

Comment:
https://github.com/slim-template/slim#trailing-and-leading-whitespace--
It's ugly as hell, but...